connect4.js

```diff
@@ -97,10 +97,12 @@ function handleClick(evt) {
   }
 
   // check for tie
-  // TODO: check if all cells in board are filled; if so call, call endGame
+  if (board.every(row => row.every(cell => cell))) {
```

Comment:
minor: might be cheaper to keep a counter of filled cells rather than checking full array everytime

Reply:
Good point. I will implement that.